src/content/general/organizations/index.md

```diff
@@ -0,0 +1,145 @@
+
+Giant Swarm management clusters provide a CRD named `Organization` (long form: `organizations.security.giantswarm.io`). An organization is defined simply by a custom resource using that CRD, which we'll call an "organization CR" here for brevity.
+
+> An organization is defined by an organization CR.
+
+Our CRD schema documentation provides details about the [Organization CRD]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}). But before you raise your eyebrows in disappointment, be warned: there isn't much to document. The single most important aspect of an organization CR is it's name. But there is more to it, of course.
+
+Once an organization CR is created, our automation ([organization-operator](https://github.com/giantswarm/organization-operator), to be precise) ensures that a namespace exists for the organization. More about that [in a minute](#namespace).
+
+## Naming conventions {#naming-conventions}
+
+Organization names (technically: organization CR names) must follow these rules:
+
+- Must be unique within the management cluster
+- Must contain at most 59 characters.
+- Has to conform the same convention as Kubernetes namespaces additionally (i. e. the [DNS label names convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)), which means:
+    - contain only lowercase alphanumeric characters or '-'
+    - start with an alphanumeric character
+    - end with an alphanumeric character
+
+Since there will be a namespace created for each org, prefixed with `org-`, we recommend against using that same prefix in the organization name, to avoid confusion.
+
+## Organization namespace {#namespace}
+
+For each organization there is a namespace in the management cluster. The namespace is named after the organization CR name, prefixed with `org-`.
+
+For example, for an organization `acme`, there is the defining organization CR named `acme`. In addition, organization-operator ensures the existence of the namespace `org-acme` in the management cluster.
+
+We recommend to place all resources belonging to an organization into the organization's namespace. Our user interfaces will work towards supporting this as a default.
+
+### Namespace utilization in different providers {#namespace-use}
+
+Giant Swarm is currently working towards making the organization's namespace the default namespace for all resources owned by the organization: clusters, node pools, apps, and more. We have reached different stages on different providers as of May 2021:
+
+- **Azure**: With the latest workload cluster releases for Azure (as of April 2021 that is v14.1.x), all cluster and node pool resources are placed in the organization namespace by default.
+
+  Resources belonging to apps deployed to workload clusters are not yet placed in the organization namespace.
```

Comment:
aren't these going into cluster namespace by default?

Reply:
btw, do we still have cluster ns and if yes should we maybe include some docs about it either in this doc or somehow linked from here?

Reply:
Yes, cluster namespace it is.

Not sure if that it documented somewhere.